GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -1547,6 +1552,7 @@ void QtQuickAdapter::processWalletAddresses(const std::string& walletId
    const auto lastAddr = addresses.at(addresses.size() - 1);
    logger_->debug("[{}] {} last address: {}", __func__, hdWalletId, lastAddr.address.display());
    addrModel_->addRow(hdWalletId, { QString::fromStdString(lastAddr.address.display())
+      , QString::fromStdString(lastAddr.index)
       , QString::fromStdString(lastAddr.index) });
```

Comment:
If this is not a typo, what's the purpose of duplicating this data? Maybe better to set address comment to empty string if it's not available?